torch/_torch_docs.py

```diff
@@ -3035,8 +3035,10 @@ def merge_dicts(*dicts):
 Creates a :class:`Tensor` from a :class:`numpy.ndarray`.
 
 The returned tensor and :attr:`ndarray` share the same memory. Modifications to
-the tensor will be reflected in the :attr:`ndarray` and vice versa. The returned
-tensor is not resizable.
+the tensor will be reflected in the :attr:`ndarray` and vice versa. 
```

Comment:
Don't change this paragraph, instead add a warning after the last paragraph and before the example. The warning should say:

"Writing to a tensor created from a read-only NumPy array is not supported and will result in undefined behavior."